a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -2462,41 +2462,47 @@ <h2>Implementations</h2>
 			possibly volunteer an implementation, please put
 			your company name, your name and your contact information.
 		</div>
+		-->
+
 
 		<p>These guidelines provide a general framework and make
 			suggestions on the display of accessibility
 			metadata. It is not a normative description of what must be
 			done. It is instructive to provide examples
 			of implementations from the community. </p>
 
-		<p>Linked below are static pages that show real-life
-			implementations. We have captured these examples from
+		<p>We have created a webpage that captures examples from
 			organization's websites that have agreed to allow us to
 			showcase the work they have done to display
 			accessibility metadata.</p>
 
-		<p>Links TBD</p>
+		<p><a href="https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/implementations/">Examples of implementations of these guidelines.</a></p>
 	</section>
 	<section id="change-log" class="informative">
 		<h2>Change log</h2>
-		
+
 		<div class="note">
-			<p>This section identifies substantive changes between versions. For a list of all changes, refer to 
+			<p>This section identifies substantive changes between versions. For a list of all changes, refer to
 				the <a
 					href="https://github.com/w3c/publ-a11y/issues?q=is%3Aissue%20state%3Aclosed%20label%3Aa11y-display-guide">issue tracker</a>.</p>
 		</div>
-		
+
 		<details open="">
-			<summary>Changes since the <a 
+			<summary>Changes since the <a
 					href="https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/guidelines/guidelines-draft-note-20250220.html">2025-02-20
 					Draft Community Group Report</a></summary>
 			<ul>
-				<li>18-Mar-2025: Fixed a typo in the descriptive statement for unknown hazards. See <a 
+				<li>18-Mar-2025: Fixed a typo in the descriptive statement for unknown hazards. See <a
 						href="https://github.com/w3c/publ-a11y/issues/674">issue 674</a>.</li>
-				<li>03-Mar-2025: Added outputs for situations where there is a mix of hazard, no hazard, and 
-					unknown hazard statements. See <a href="https://github.com/w3c/publ-a11y/issues/578">issue 
+				<li>03-Mar-2025: Added outputs for situations where there is a mix of hazard, no hazard, and
+					unknown hazard statements. See <a href="https://github.com/w3c/publ-a11y/issues/578">issue
 						587</a>.</li>
 				<li>28-Feb-2025: Fixed missing space at the end of the <code>conformance-certifier</code> string.</li>
+<li>29-Mar-2025: Fixed language describing accessible rich content. See <a
+						href="https://github.com/w3c/publ-a11y/issues/671">issue 671</a>.</li>
+			<li>29-Mar-2025: Fixed example 12 that was using incorrect strings for filtering. See <a
+						href="https://github.com/w3c/publ-a11y/issues/673">issue 673</a>.</li>
+						<li>29-Mar-2025: Revised Implementation section to point to an external page rather than embedding static images of examples.</li>
 			</ul>
```

Comment:
For the change logs, it's always newest entries first, so the entries you added should be moved to the top.